projects/js-packages/publicize-components/src/utils/test-mocks.js

```diff
@@ -0,0 +1,79 @@
+import { jest } from '@jest/globals';
```

Comment:
This extreme level of mocking is what makes me hate unit tests in favor of E2E tests.

Reply:
Yeah, agreed. On the other hand it has the advantage of being quick and good for testing the logic of each component. We should have both!